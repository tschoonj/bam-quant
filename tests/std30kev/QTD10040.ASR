File: C:\xrfdaten\2014KW30\chris\std30kev\std10040.spe
Start: 22.07.2014 10:10:39
Stop: 22.07.2014 10:12:19
BESSY: 298.0045841 298.8194486
IONI: 1.974243E-11 1.971507E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 74602
Run: 100000
Dead: 25
ICR: 83030
OCR: 0
Accumulated BESSY: 59691.2695165
Accumulated Ioni: 3.94140291E-09
N Accumulation: 200
Normfactor:  6.8018757e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Zr
exp_x 152.720397949219 152.7204
exp_y 122.465805053711 122.4658
exp_z 53.8899993896484 53.8902
exp_w 135 135.0001
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  40   1     15.746   4957897.      2227.      88.56
