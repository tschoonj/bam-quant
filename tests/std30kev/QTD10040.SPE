$DATA:
0 2000
17394
11272
5060
1639
369
72
37
11
4
6
2
2
3
0
2
8
193
1584
1990
1954
1878
1768
1796
1690
1676
1590
1609
1694
1607
1616
1603
1650
1559
1609
1536
1553
1645
1636
1531
1486
1481
1525
1475
1379
1361
1342
1318
1271
1229
1149
1114
1094
1017
1047
987
982
973
879
844
821
862
751
782
797
775
692
696
709
703
650
700
687
702
683
701
675
602
581
605
541
535
536
553
519
527
520
511
520
529
497
500
522
485
503
509
526
551
466
504
497
504
447
496
474
477
482
479
465
468
448
465
413
394
430
419
437
414
428
421
428
385
437
436
389
404
405
439
360
389
361
403
364
416
362
392
364
354
361
404
369
364
370
355
364
387
353
372
365
379
398
341
328
389
360
371
336
385
332
356
353
368
353
323
311
341
344
329
323
320
361
365
307
346
342
310
328
335
336
354
301
312
335
304
341
326
336
314
302
294
317
305
326
286
294
331
319
332
298
319
288
355
316
285
316
311
324
306
303
311
283
296
305
300
289
282
292
285
261
283
294
310
295
264
321
293
295
293
284
275
327
314
283
295
309
290
280
311
283
277
296
297
258
309
297
294
255
288
304
266
255
267
280
251
299
259
299
290
270
305
285
260
306
253
278
267
300
282
294
298
309
289
270
276
251
278
255
284
255
261
283
287
260
285
264
256
254
254
300
245
259
273
288
270
275
262
295
256
294
256
252
250
299
268
267
267
273
245
248
259
280
272
293
337
326
385
448
416
454
437
404
375
304
313
280
270
267
257
268
252
288
260
250
260
262
282
259
265
263
295
248
289
257
239
261
287
250
246
287
271
237
293
258
295
285
245
267
256
264
238
248
273
235
256
255
280
241
265
236
260
268
259
252
258
256
268
271
215
251
269
249
281
216
248
241
261
256
275
229
255
255
235
249
241
247
251
285
254
251
278
239
250
227
254
262
272
259
238
226
263
264
240
239
251
247
271
248
275
259
237
248
241
260
252
225
244
255
253
262
261
253
262
264
258
269
258
239
258
235
248
259
244
248
272
281
255
260
259
281
265
247
246
260
263
240
257
287
267
303
273
281
301
255
267
235
275
263
254
237
259
230
258
273
242
212
288
261
256
239
246
238
250
238
263
248
241
226
275
271
276
218
294
260
262
236
282
258
289
244
257
253
258
231
276
305
247
242
261
274
281
261
249
251
271
288
268
286
256
279
266
286
288
299
276
316
303
266
283
287
275
269
269
279
301
282
290
314
292
280
275
279
261
264
296
258
296
278
295
292
285
253
257
288
299
305
280
259
283
298
272
324
307
279
292
273
285
293
297
290
298
283
319
280
313
267
294
277
314
291
279
312
302
288
311
304
304
309
307
316
296
323
331
311
314
307
321
330
342
328
318
347
327
344
324
320
353
335
348
327
345
318
371
356
330
321
332
360
330
358
315
379
342
350
374
378
371
354
350
415
368
362
406
373
369
389
376
365
364
383
365
388
357
417
402
383
432
406
412
393
409
462
446
418
481
428
414
413
425
429
469
428
445
435
445
446
466
442
471
430
470
443
475
468
408
484
444
450
454
471
486
514
554
536
524
574
591
597
616
637
678
704
759
913
843
795
799
855
799
830
751
731
711
645
652
699
631
701
631
667
650
685
729
717
697
773
780
784
784
893
786
855
870
926
930
929
964
1013
1009
1111
1105
1172
1161
1218
1291
1277
1406
1456
1436
1507
1565
1580
1564
1675
1589
1693
1772
1855
1840
1917
1951
2035
2101
2082
2098
2263
2323
2329
2455
2613
2695
2968
3105
3529
3901
4904
6134
8476
12102
18152
27586
41100
61013
85926
118946
159035
203698
251656
297850
340720
375273
394898
400975
389128
363779
324357
276462
225396
176541
131227
94041
64344
42466
26835
16578
10149
6169
3996
2695
2061
1605
1383
1221
1150
1028
1005
940
888
847
838
835
827
759
724
755
687
725
735
729
748
719
745
745
676
774
688
759
759
705
752
746
764
772
826
765
780
804
844
841
838
849
860
866
941
915
993
1020
1055
1103
1144
1160
1303
1228
1276
1376
1401
1413
1453
1545
1683
1820
2096
2426
3091
4077
5834
8421
12105
17365
24179
32671
41752
52632
63382
72341
79758
82502
83291
79284
73086
63881
54330
44355
34688
27414
21347
17492
14978
13708
13099
12922
12714
12200
11510
10466
9158
7595
6054
4697
3483
2470
1588
1083
680
418
227
126
114
81
64
48
42
53
39
33
34
42
28
33
23
30
29
28
31
20
20
24
21
29
21
27
19
16
17
22
13
15
15
21
20
10
13
13
13
10
16
12
18
20
13
10
12
15
13
13
17
14
13
17
7
14
12
15
9
12
12
7
13
11
17
6
15
12
9
4
7
6
10
16
10
7
7
11
11
5
8
7
9
10
6
11
8
11
7
13
8
10
7
7
8
9
6
14
6
4
7
8
5
5
8
14
4
5
14
7
11
9
5
8
8
7
4
7
5
11
11
10
9
3
11
5
8
7
11
11
7
10
7
6
9
4
6
11
6
4
11
9
3
10
7
8
7
10
5
5
3
4
6
8
10
9
2
7
7
5
3
11
6
7
3
7
5
7
7
6
6
10
8
15
8
6
6
8
7
3
3
3
5
5
9
7
9
11
3
15
13
6
8
11
8
7
7
8
11
7
7
10
3
9
4
8
7
8
2
3
4
7
10
7
6
6
7
8
4
6
4
5
9
3
5
3
5
4
3
14
5
3
9
5
6
9
5
7
9
9
4
5
5
8
8
7
11
9
13
7
8
5
11
14
5
9
4
7
8
5
12
9
7
5
3
6
8
4
7
2
14
5
8
6
9
4
8
6
8
3
2
2
6
8
9
9
10
12
5
12
7
8
3
4
9
6
14
4
6
6
4
8
6
7
8
9
8
7
9
5
5
3
10
8
6
10
9
10
17
9
9
13
16
16
24
32
35
54
61
79
109
111
165
189
201
232
248
286
309
306
336
343
370
370
382
423
416
447
395
416
397
354
321
239
251
179
146
110
75
62
59
39
19
17
20
9
8
2
7
7
11
4
7
10
7
13
5
7
8
7
4
3
13
4
4
3
7
10
7
8
7
10
5
9
4
2
4
3
4
4
10
9
3
6
5
4
4
5
4
6
9
7
5
6
5
7
7
4
8
7
3
10
7
3
4
4
5
3
6
7
3
8
6
3
4
4
6
6
2
11
8
4
5
2
8
3
9
2
6
6
3
4
10
5
4
4
5
12
5
6
3
9
6
3
5
2
4
6
5
6
8
8
8
6
11
6
6
3
5
8
8
8
6
8
10
7
9
8
7
11
10
8
9
13
23
20
32
30
42
42
53
69
80
99
84
98
102
114
121
85
93
89
85
69
52
38
38
32
16
22
16
12
7
6
9
10
10
6
15
19
12
15
17
16
21
22
25
30
35
20
20
22
30
23
16
20
7
16
10
13
8
10
6
8
13
5
7
7
6
3
8
10
13
8
13
15
9
17
9
7
15
14
8
6
12
8
8
8
10
12
10
14
12
9
6
18
15
10
8
7
11
7
7
12
20
8
16
14
17
10
18
16
14
15
20
8
15
15
20
24
18
25
19
23
21
26
19
28
25
23
20
23
23
22
35
30
32
34
40
22
36
53
49
54
78
97
122
148
172
259
343
360
468
567
653
800
940
1038
1293
1330
1489
1544
1693
1705
1797
1764
1748
1542
1503
1406
1268
1089
914
868
697
574
462
352
310
207
128
133
87
75
51
25
42
34
21
25
21
15
17
10
13
19
9
12
24
18
16
18
15
15
15
11
13
18
12
20
20
20
19
17
11
14
29
22
18
15
20
23
23
26
24
15
31
23
35
28
30
34
28
36
33
27
36
44
52
64
60
63
90
91
136
171
163
222
281
288
337
428
467
591
611
558
684
718
759
700
733
706
684
633
605
533
484
484
372
359
289
265
235
207
182
155
136
115
107
99
96
87
52
65
51
26
44
28
24
22
21
23
15
17
7
14
14
10
7
15
8
14
15
17
13
14
13
11
16
9
10
8
9
11
11
12
8
6
11
12
12
18
22
8
14
25
15
18
18
16
18
27
20
28
29
31
32
35
27
37
41
35
60
58
73
79
75
87
80
89
91
89
87
108
83
90
97
114
92
93
61
68
69
66
73
62
51
42
27
37
24
24
26
25
13
15
15
14
15
10
6
2
7
6
5
4
3
0
0
5
3
1
2
3
0
0
0
1
1
0
0
0
0
0
0
0
0
1
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
1
0
0
0
0
1
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
0
1
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10040.spe
Start: 22.07.2014 10:10:39
Stop: 22.07.2014 10:12:19
BESSY: 298.0045841 298.8194486
IONI: 1.974243E-11 1.971507E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 74602
Run: 100000
Dead: 25
ICR: 83030
OCR: 0
Accumulated BESSY: 59691.2695165
Accumulated Ioni: 3.94140291E-09
N Accumulation: 200
Normfactor:  6.8018757e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Zr
 
exp_x 152.720397949219 152.7204
exp_y 122.465805053711 122.4658
exp_z 53.8899993896484 53.8902
exp_w 135 135.0001
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  40   1     15.746   4957897.      2227.      88.56
