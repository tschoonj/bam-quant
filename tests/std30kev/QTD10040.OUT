     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:49  บ   
     บ  Spectrum: QTD10040.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        4 iterations done    บ   
     บ  ChiSquare =   128.6   last change =   .02%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -57.7 ๑   100.0   บ     -57.7 ๑      .6   บ   
     บ  GAIN (eV/ch)  บ     20.245 ๑   2.000   บ    20.245 ๑    .001   บ   
     บ det NOISE (eV) บ     -113.2 ๑    40.0   บ    -113.2 ๑    -3.7   บ   
     บ  FANO factor   บ       .180 ๑    .050   บ      .180 ๑    .002   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Zr-Ka                           4957897. ๑   2227.           บ   
     บ    KA1        15.775    .65677     3256208. ๑   1463.    103.0  บ   
     บ              782.062    218.70         722.           9.88E-01  บ   
     บ    KA2        15.691    .34323     1701689. ๑    765.     96.1  บ   
     บ              777.913    217.96         741.           9.88E-01  บ   
     บ                                                                 บ   
     บ  2 Zr-Kb                           1139340. ๑   1068.           บ   
     บ    KB1        17.663    .87511      997042. ๑    935.    105.0  บ   
     บ              875.321    234.71         424.           9.81E-01  บ   
     บ    KB2        17.970    .12489      142297. ๑    135.     10.7  บ   
     บ              890.485    237.21         386.           9.79E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                           48141. ๑    222.           บ   
     บ    L1O3       31.492    .66115       31846. ๑    148.      7.5  บ   
     บ             1558.426    329.02         268.           5.82E-01  บ   
     บ    L1M5       33.409    .26597       12811. ๑     61.      4.0  บ   
     บ             1653.108    340.03         249.           5.27E-01  บ   
     บ    L1M4       33.716    .03796        1828. ๑     17.      1.4  บ   
     บ             1668.273    341.76         235.           5.19E-01  บ   
     บ    L1N5       35.326    .02675        1288. ๑     12.      9.1  บ   
     บ             1747.791    350.70         117.           4.78E-01  บ   
     บ    L1N4       35.633    .00764         368. ๑      9.      3.5  บ   
     บ             1762.955    352.38          85.           4.71E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
